go/buildhelper/client.go

```diff
@@ -0,0 +1,96 @@
+package buildhelper
```

Comment:
@otherview Can this whole package get a readme too? I'm confused about what makes something belong in this package.

Reply:
@otherview Am I right to think that this is all code used to support the integration tests? Should we move it under the `integration` package?

Reply:
> @otherview Can this whole package get a readme too? I'm confused about what makes something belong in this package.

Yeah that's a good question - This package is just to make sure changes are isolated enough so to not have merge conflicts. That was the main concern.

It needs to be cleaned up, let me work a bit on that.


> @otherview Am I right to think that this is all code used to support the integration tests? Should we move it under the integration package?

Not all. This will support connecting L1 nodes, so I wouldn't move everything to the `integration` package.
Unless the `integration` package also deals with connecting L1 to L2 🤔 ? I thought it was only simulation/test related code ?